File: Pattern/pattern2/pattern.go
package main

import "fmt"

func pattern(x int) {
	for i := 0; i < x; i++ {
		for j := 0; j <= i; j++ {
			fmt.Print("*")
		}
		fmt.Println("")
	}
}

func main() {
	pattern(6)
}

File: Pattern/pattern2/Pattern.java
package Pattern.pattern2;

public class Pattern {
    public static void main(String[] args) {
        pattern(5);
    }

    public static void pattern(Integer x) {
        for (int i = 0; i <= x; i++) {
            for (int j = 0; j < i; j++) {
                System.out.print("*");
            }
            System.out.println("");
        }
    }
}
